entrypoint.sh

```diff
@@ -6,6 +6,8 @@ source=${SOURCE:-.}
 dryrun=${DRY_RUN:-false}
 new_version=${NEW_VERSION:-""}
 
+git config --global --add safe.directory /github/workspace 
```

Comment:
I think you're missing a dot here. This should be
```suggestion
git config --global --add safe.directory /.github/workspace 
```